<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceTestPage</class>
 <widget class="QWidget" name="MaintainceTestPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
font:20px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
color:white;
background-color:blue;
}
QPushButton
{
	border-radius:8px;
font: 20px, &quot;微软雅黑&quot;;color: white;background-color: rgb(0, 128, 128);border-top: 2px solid white;border-left: 2px solid white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>520</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>530</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="MTempM2lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>15</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MTempM1lbl">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>15</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MTempMP1lbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>15</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="MTempMP2lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>15</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_9">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>电机温度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>BC压
(Bar)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC11lbl">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3.4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>95</y>
     <width>800</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC12lbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP12lbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP11lbl">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM12lbl">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM11lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM21lbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPM22lbl">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP21lbl">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPMP22lbl">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC21lbl">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BCPTC22lbl">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>速度选择</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Speedlbl">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>km/h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>140</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>加速度(+)减速度(-)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ASpeedlbl">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>99.99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_7">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>140</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>操作级位</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="levellbl">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>150</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>加速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>减速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>0－&gt;40km/h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_13">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="A40Speedlbl">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_14">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>0－&gt;80km/h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="A80Speedlbl">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>220</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_15">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="InitSpeedlbl">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_16">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>km/h</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="AveSpeedlbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_17">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>m/s²</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="BrakeDistancelbl">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_18">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>320</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>m</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Braketimelbl">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:24px  &quot;微软雅黑&quot;;
color:rgb(23,176,176);
background-color:black;</string>
   </property>
   <property name="text">
    <string>0.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_19">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>320</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>s</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_20">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
font:16px;</string>
   </property>
   <property name="text">
    <string>起始速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_21">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
font:16px;</string>
   </property>
   <property name="text">
    <string>平均减速度</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_22">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
font:16px;</string>
   </property>
   <property name="text">
    <string>制动距离</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl1_23">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
font:16px;</string>
   </property>
   <property name="text">
    <string>制动时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="StartTestBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>开始测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StopTestBtn">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>结束测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="Teststatuslbl">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>481</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
color:black;</string>
   </property>
   <property name="text">
    <string>测试未进行</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Teststatuslbl_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>731</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black; font:16px;</string>
   </property>
   <property name="text">
    <string>列车静止时，点击“开始测试”按钮后，显示屏显示“测试进行中”。
司控器手柄推至牵引后，进行加速度测试。列车速度大于零后，手柄推至制动位置，进行减速度测试。
测试结束后，点击“测试结束”按钮。</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lbl1_9</zorder>
  <zorder>lbl1_10</zorder>
  <zorder>label</zorder>
  <zorder>returnBtn</zorder>
  <zorder>MTempM2lbl</zorder>
  <zorder>MTempM1lbl</zorder>
  <zorder>MTempMP1lbl</zorder>
  <zorder>MTempMP2lbl</zorder>
  <zorder>lbl1</zorder>
  <zorder>lbl1_2</zorder>
  <zorder>BCPTC11lbl</zorder>
  <zorder>BCPTC12lbl</zorder>
  <zorder>BCPMP12lbl</zorder>
  <zorder>BCPMP11lbl</zorder>
  <zorder>BCPM12lbl</zorder>
  <zorder>BCPM11lbl</zorder>
  <zorder>BCPM21lbl</zorder>
  <zorder>BCPM22lbl</zorder>
  <zorder>BCPMP21lbl</zorder>
  <zorder>BCPMP22lbl</zorder>
  <zorder>BCPTC21lbl</zorder>
  <zorder>BCPTC22lbl</zorder>
  <zorder>lbl1_3</zorder>
  <zorder>Speedlbl</zorder>
  <zorder>lbl1_4</zorder>
  <zorder>lbl1_5</zorder>
  <zorder>lbl1_6</zorder>
  <zorder>ASpeedlbl</zorder>
  <zorder>lbl1_7</zorder>
  <zorder>levellbl</zorder>
  <zorder>lbl1_8</zorder>
  <zorder>lbl1_11</zorder>
  <zorder>lbl1_12</zorder>
  <zorder>lbl1_13</zorder>
  <zorder>A40Speedlbl</zorder>
  <zorder>lbl1_14</zorder>
  <zorder>A80Speedlbl</zorder>
  <zorder>lbl1_15</zorder>
  <zorder>InitSpeedlbl</zorder>
  <zorder>lbl1_16</zorder>
  <zorder>AveSpeedlbl</zorder>
  <zorder>lbl1_17</zorder>
  <zorder>BrakeDistancelbl</zorder>
  <zorder>lbl1_18</zorder>
  <zorder>Braketimelbl</zorder>
  <zorder>lbl1_19</zorder>
  <zorder>lbl1_20</zorder>
  <zorder>lbl1_21</zorder>
  <zorder>lbl1_22</zorder>
  <zorder>lbl1_23</zorder>
  <zorder>StartTestBtn</zorder>
  <zorder>StopTestBtn</zorder>
  <zorder>Teststatuslbl</zorder>
  <zorder>Teststatuslbl_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
